### PR TITLE
Added few more anime apps and website

### PR DESCRIPTION
**Pull request by guywhogeek**
Apps -
* Animania
* Popcorn Time

Website - 
* Viewster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +64,12 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [viewster.com](https://www.viewster.com/)
```
Comment (anshumanv) [inline README.md]: The content seems to limited and I see some issues with it.

image


Can you please remove it instead?

File: README.md [hunk 2]
```diff
@@ -64,9 +64,12 @@ A curated list of anime streaming/downloading websites.
...
 
 ### Anime (mobile app)
 * [Anime HD Watch - Kissanime](https://play.google.com/store/apps/details?id=com.pg.kissanimenew&hl=en)
+* [#Animania](http://www.mobi24.net/#anime)
+* [Popcorn Time](https://popcorntime.sh/)
```
Comment (anshumanv) [inline README.md]: Does it provide animes as well? Please share ss. :smile:

File: README.md (hunk 1 above)
Comment (guywhogeek) [reply README.md]: Content does seem to be pretty limited. I will rule this one out.

File: README.md (hunk 2 above)
Comment (guywhogeek) [reply README.md]: Screenshot for animania is available on the mentioned link (http://www.mobi24.net/#anime).

For popcorn time any content available on torrent can be streamed. Thus, if the show is available on torrent, it is available on popcorn time.

Comment (anshumanv) [reply README.md]: > Screenshot for animania is available on the mentioned link (http://www.mobi24.net/#anime).
> 
> For popcorn time, any content available on torrent can be streamed. Thus, if the show is available on torrent, it is available on popcorn time.

So popcorn seems like a generic solution and not centered to anime so I'm not sure if this is a right pick for this list, please remove it as well. :smile:

**Comment by guywhogeek** [comment]
Done

**Comment by anshumanv** [comment]
Thanks! :tada:

**Pull request merged**